views/islandora_usage_stats.views_default.inc

```diff
@@ -546,6 +546,8 @@ function islandora_usage_stats_views_default_views() {
   $handler->display->display_options['fields']['term']['id'] = 'term';
   $handler->display->display_options['fields']['term']['table'] = 'islandora_usage_stats_top_searches';
   $handler->display->display_options['fields']['term']['field'] = 'term';
+  $handler->display->display_options['fields']['term']['alter']['make_link'] = TRUE;
```

Comment:
The link isn't showing for me... even after clearing the cache about 90 times.

Am I missing something, or is the code missing something?